Sources/SecureXPC/Server/XPCServiceServer.swift

```diff
@@ -55,4 +58,17 @@ internal class XPCServiceServer: XPCServer {
 		// XPC services are application-scoped, so we're assuming they're inheritently safe
 		true
 	}
+
+    public override var endpoint: XPCServerEndpoint {
+        guard let connection = Self.connection else {
+            fatalError("You can only create an `endpoint` for an XPCServiceServer after starting it with `start()`.")
+        }
+
+        let endpoint = xpc_endpoint_create(connection)
+        return XPCServerEndpoint(
+            kind: .xpcServiceClient,
+            serviceName: "TODO: implement me",
```

Comment:
> In case you didn't already know, this is the name of the bundle so Bundle.main.bundleIdentifier should suffice

Perfect!

> Although it can in theory be nil so perhaps it makes sense to access and store this during initialization

This would be great. It would allow our `XPCServer`s (of any type) to all have a name. (except for anonymous ones, of course)